onnxruntime/core/providers/webgpu/tensor/concat.h

```diff
@@ -24,6 +24,20 @@ class ConcatProgram final : public Program<ConcatProgram> {
   size_t axis_;
 };
 
+class ConcatProgramSingle final : public Program<ConcatProgramSingle> {
```

Comment:
Use `ConcatSingleProgram` or something similar to it. we usually name a program class like `XXXXProgram`